x-pack/plugins/observability_onboarding/public/components/app/custom_logs/wizard/api_key_banner.tsx

```diff
@@ -94,7 +94,11 @@ export function ApiKeyBanner({
                 iconType="copyClipboard"
                 onClick={copy}
                 color="success"
-                style={{ backgroundColor: 'transparent' }}
+                css={{
+                  '> svg.euiIcon': {
```

Comment:
(nit) the only challenge i see targeting selectors from library which we do not control directly is, if in a future upgrade they change the selector, this could break

Reply:
What would you recommend? The original issue was related to a icon that was cropped, I looked for the root of this and discovered this rounded style it's embedded in a kibana theme.
I get your point but I would argue that even if this break in the future it's just a rounding style, imo nothing major.